probe-rs/targets/CC13XX_CC26XX_Series.yaml

```diff
@@ -246,3 +442,55 @@ flash_algorithms:
+  load_address: 0x20000020
+  pc_init: 0x1
+  pc_uninit: 0x12d
+  pc_program_page: 0x1c7
+  pc_erase_sector: 0x16f
+  pc_erase_all: 0x235
+  data_section_offset: 0x20002264
+  rtt_location: 0x20001e24
+  flash_properties:
+    address_range:
+      start: 0x0
+      end: 0x100000
+    page_size: 0x800
+    erased_byte_value: 0xff
+    program_page_timeout: 1000
+    erase_sector_timeout: 2000
+    sectors:
+    - size: 0x800
+      address: 0x0
+  cores:
+  - main
+- name: cc13x4-cc26x4-flash-ccfg
```

Comment:
@bugadani, I could use your opinion here:

These chips have a second region of flash called the CCFG that is at a different base address from the main flash.
However, the same flash algorithm can program both, so what I did is made two different algorithms with the same instructions but different `flash_properties`. It feels like a hack....is there a better way?

Reply:
I believe this is the best you can do currently.